packages/backend/src/config/config.local.ts

```diff
@@ -51,21 +51,21 @@ export function getLocalConfig(): Config {
         workQueueWorkers: 1,
         projects: {
           ethereum: {
-            callsPerMinute: 60 / 5,
+            callsPerMinute: 60 * 10,
             url: getEnv(
               'ACTIVITY_ETHEREUM_URL',
               'https://eth-mainnet.alchemyapi.io/v2/demo',
             ),
           },
           optimism: {
-            callsPerMinute: 60 / 5,
+            callsPerMinute: 60 * 10,
             url: getEnv(
               'ACTIVITY_OPTIMISM_URL',
               'https://mainnet.optimism.io/',
             ),
           },
           arbitrum: {
-            callsPerMinute: 60 / 5,
+            callsPerMinute: 60 * 10,
```

Comment:
These has been increased to speed up latest block bin search locally - otherwise we would have maximum 12 calls per minute.